test/nn/models/test_revgnn.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+import torch
+
+from torch_geometric.nn import GroupAddRev, SAGEConv
+from torch_geometric.nn.dense.linear import Linear
+
+num_groups = [2, 4, 8, 16]
+
+
+@pytest.mark.parametrize('num_groups', [2, 4, 8, 16])
+def test_revgnn(num_groups):
+    hidden_channels = 32
+    x = torch.randn(4, 32)
+    edge_index = torch.tensor([[0, 1, 2, 3], [0, 0, 1, 1]])
+    seed_gnn = SAGEConv(
```

Comment:
Nit: seed_gnn_layer maybe better, since here it's just a GNN layer not a full network